C/services/common/plugin_manager.cpp

```diff
@@ -569,7 +569,7 @@ void PluginManager::getInstalledPlugins(const string& type,
 			// Can not open specified dir path
 			char msg[128];
 			char* ret = strerror_r(errno, msg, 128);
-			logger->error("Can not access plugin directory %s: %s",
+			logger->warn("Can not access plugin directory %s: %s",
```

Comment:
@pintomax You have made a comment on JIRA
```
changed log level to debug (Fledge side)
```

Reply:
@ashish-jabble Fixed comment: it's now warning in the JIRA comment